### PR TITLE
Carding AIs now takes time.

### PR DESCRIPTION
**Pull request by lewcc**
## What Does This PR Do
Adds a delay of 5 seconds when carding living AIs. It's no longer an instantaneous click, unless they're dead.

## Why It's Good For The Game
Downloading an AI to a card is kind of a big deal. It's one click to end a malf AI's round, and means that AIs can't really make any form of escape.
Put another way, this should make fighting an AI, especially one who's in a mobile core or has friends, a bit more dynamic.

## Testing
Carded an AI, waited five seconds before it was transferred to the card. Clicked the core again to instantly put it back.

## Changelog
:cl:
tweak: Carding a living AI now takes five seconds.
/:cl:

### CODE REVIEW

**Comment by matttheficus** [comment]
No more teleport/jaunter into Carding. I like it.

**Review by PidgeyThePirate — APPROVED** [review]
This looks okay. Simple, clear.

**Review by Burzah — APPROVED** [review]

File: code/modules/mob/living/silicon/ai/ai_mob.dm [hunk 1]
```diff
@@ -1323,6 +1323,14 @@ GLOBAL_LIST_INIT(ai_verbs_default, list(
 		if(!mind)
 			to_chat(user, "<span class='warning'>No intelligence patterns detected.</span>")//No more magical carding of empty cores, AI RETURN TO BODY!!!11
 			return
+
+		if(stat != DEAD)
+			to_chat(user, "<span class='notice'>Beginning active intelligence transfer: please wait.</span>")
+
+			if(!do_after(user, 5 SECONDS, target = src) || !Adjacent(user))
```
Comment (Sirryan2002) [inline code/modules/mob/living/silicon/ai/ai_mob.dm:1330]: make this a do_after_once

Comment (S34NW) [reply code/modules/mob/living/silicon/ai/ai_mob.dm:1330]: @lewcc

Comment (lewcc) [reply code/modules/mob/living/silicon/ai/ai_mob.dm]: done!

**Review by S34NW — APPROVED** [review]

**Pull request merged**